internal/eval/store_eval.go

```diff
@@ -473,6 +473,62 @@ func evalZADD(args []string, store *dstore.Store) *EvalResponse {
 	}
 }
 
+func parseScore(input string) (float64, error) {
+	switch input {
+	case "-inf":
+		return math.Inf(-1), nil
+	case "+inf":
+		return math.Inf(1), nil
+	default:
+		return strconv.ParseFloat(input, 64)
+	}
+}
```

Comment:
Isn't this a utility method, and should be moved to util folder? Looks like a common functionality which may be used in some other commands.

Reply:
Resolved